incompatibility/incomp-scanning_test.go

```diff
@@ -85,7 +87,7 @@ func TestIncompatibilityExistence(t *testing.T) {
 				_, searchErr :=
 					findNode(node.Content[0], incomp.GetTokenPath()...)
 				if searchErr != nil {
-					tt.Errorf(searchErr.Error())
+					tt.Errorf(searchErr.Error(), fmt.Sprintf("%+v\n", incomp))
```

Comment:
Do we want this in the form of want, got?

Reply:
think reporting the missing incompatibility is enough, this test doesn't really include a want, what we are looking for here is that there exists such a node associated with the reported path in the incompatibility